product_docs/docs/biganimal/release/using_cluster/extensions.mdx

```diff
@@ -69,7 +70,7 @@ EDB extensions:
 
 ## Installing extensions
 
-Use the [`CREATE EXTENSION`](https://www.postgresql.org/docs/current/sql-createextension.html) command to install extensions, except for the EDB Postgres Tuner (pg_tuner) extension. You enable edb_pg_tuner on the **DB Configuration** tab of the Create or Edit Cluster page of the BigAnimal portal.
+Use the [`CREATE EXTENSION`](https://www.postgresql.org/docs/current/sql-createextension.html) command to install most extensions. You must enable certain extensions, including the EDB Postgres Tuner (pg_tuner) extension and PostGIS on the **DB Configuration** tab of the Create or Edit Cluster page of the BigAnimal portal.
```

Comment:
```suggestion
Use the [`CREATE EXTENSION`](https://www.postgresql.org/docs/current/sql-createextension.html) command to install most extensions. You must enable certain extensions, including the EDB Postgres Tuner (pg_tuner) extension and PostGIS. Enable pg_tuner on the **DB Configuration** tab of the Create or Edit Cluster page of the BigAnimal portal. Enable PostGIS on the **Additional Settings** tab.
```

@amrita-suresh @djw-m All good now?

Reply:
There doesn't seem to be a simple Enable pg_tuner on DB Configuration:

<img width="868" alt="Screenshot 2023-11-14 at 17 41 20" src="https://github.com/EnterpriseDB/docs/assets/126472455/ef8937e4-cc97-46fa-9557-2850d554e4c3">

I suggest more guidance on that issue. Apart from that, it's fine.